packages/plugin-ext/src/main/browser/notebooks/renderers/output-webview-internal.ts

```diff
@@ -313,11 +313,20 @@ export async function outputWebviewPreload(ctx: PreloadContext): Promise<void> {
         private onRenderCompleted(): void {
             // we need to check for all images are loaded. Otherwise we can't determine the correct height of the output
             const images = Array.from(document.images);
+            console.log(images);
+            images?.forEach(img => img.onresize = e => console.log(e));
```

Comment:
Did you forgot to remove the debug logging?

Reply:
ups yes thanks for pointing that out